TFNetworkLayer.py

```diff
@@ -895,6 +897,8 @@ def batch_norm(self, data,
     :param tf.Tensor sample_variance:
     :param tf.Tensor gamma:
     :param tf.Tensor beta:
+    :param bool masked_time: flatten and mask input tensor
+    :param tf.DType dtype: data type, different from default when testing
```

Comment:
`dtype` can also be of type `str`, right? Or probably that is even the usual case which you often have. You should write that, i.e. `:param tf.DType|str dtype:`.

But anyway, I don't quite understand why you need that? It should not have an effect. Also, it's not clear from the documentation and also from the parameter name what this is really used for. Please remove this if this is not really needed.

Reply:
For some reason numpy and tf calculate mean differently (diff is about e-4 or e-5). The calculations are more accurate with float64 type. So, I need that to do tests and assert on equal results.

Reply:
No, you do not need that. Do not assert on equal results anyway. You should use `numpy.testing.assert_almost_equal`.

Reply:
No, you do not need that. Use `numpy.testing.assert_almost_equal`.